src/builder/instruction-builder.ts

```diff
@@ -58,4 +58,14 @@ export class InstructionBuilder {
     const deleteIx = this.program.instruction.deleteFlow(deleteContext);
     return { instructions: [deleteIx], signers: [] };
   }
+
+  buildDepositFeeInstruction(from: PublicKey, to: PublicKey, amount: number) {
```

Comment:
can we please rename this to **buildSystemTransferInstruction** since at this level, it is has no context of the calling methods. Deposit fee is only one way of using this instruction.